src/core/effects/qgsgloweffect.cpp

```diff
@@ -176,7 +176,14 @@ void QgsGlowEffect::readProperties( const QgsStringMap &props )
 
   //attempt to create color ramp from props
   delete mRamp;
-  mRamp = QgsGradientColorRamp::create( props );
+  if ( props.contains( QStringLiteral( "schemeName" ) ) )
```

Comment:
This check looks a bit fragile - can we instead have a proper property for the ramp type?